### PR TITLE
Fix: Use localhost as default devHost on windows

### LINKED ISSUE #257: Don't use 0.0.0.0 as default devHost on windows
Windows doesn't resolve 0.0.0.0 as localhost/127.0.0.1

[process.platform](https://nodejs.org/api/process.html#process_process_platform) will help us

### PR DESCRIPTION
**Pull request by thangngoc89**
Close #257 

Waiting for appveyor (don't have access to a windows machine atm)

### CODE REVIEW

**Comment by MoOx** [comment]
See https://github.com/MoOx/statinamic/issues/257#issuecomment-192954430

**Comment by MoOx** [comment]
Can you just simply open localhost by default without any platform distinction?

**Comment by thangngoc89** [comment]
Done. Easier than I thought

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+- Changed: Use localhost as default address to open new browser tab (for Windows
+  compatible since Windows doesn't resolve 0.0.0.0 as localhost/127.0.0.1)
```
Comment (MoOx) [inline CHANGELOG.md]: compatibiity*

File: src/builder/server.js [hunk 2]
```diff
@@ -204,7 +204,7 @@ export default (webpackConfig, options = {}) => {
     const href = `http://${ devHost }:${ devPort }${ config.baseUrl.pathname }`
     log(`Dev server started on ${ href }`)
     if (config.open) {
-      opn(href)
+      opn(href.replace("0.0.0.0", "localhost"))
```
Comment (MoOx) [inline src/builder/server.js]: href.replace(devHost, "localhost")

Comment (thangngoc89) [inline src/builder/server.js]: I can't think of any harm when doing this. But this means user can't override default open host.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,3 +1,5 @@
+- Changed: Use localhost as default address to open new browser tab (for Windows
+  compatibiity since Windows doesn't resolve 0.0.0.0 as localhost/127.0.0.1)
```
Comment (MoOx) [inline CHANGELOG.md]: compatibility*

Comment (MoOx) [inline CHANGELOG.md]: :D

Comment (thangngoc89) [inline CHANGELOG.md]: Fixed. Haha

**Comment by MoOx** [comment]
Need rebase.

**Comment by thangngoc89** [comment]
Rebased. Conflict in CHANGELOG again. I'll merge it when tests passed

**Pull request merged**